.github/workflows/main.yaml

```diff
@@ -252,9 +253,9 @@ jobs:
         id: wxwidgets-cache
         run: |
           if [ -d wxWidgets ]; then
-            echo "::set-output name=cache-hit::true"
+            echo "cache-hit=true" >> $GITHUB_OUTPUT
```

Comment:
Maybe we can just use the Cache wxWidgets `cache-hit` results? I've never seen it be wrong and if we do see it we should probably report it upstream to fix it anyway.